src/content/docs/workflow_blogposts.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Workflow: Blog posts"
+meta_title: "Workflow: Blog posts"
+description: "Workflow for request and submit a blog post"
+draft: false
+---
+
+## Request Blog Post Workflow
+
+**For the communications team**
+
+### 1. Blog Post Request Initiation
+
+* **Who:** Any team member
+* **Where:** GitHub ([PyLadiesCon repo](https://github.com/global-conference] – Pull request)
+
+Create a Pull Request entitled: `Blog Post: [Proposed Title]`
+and assign to: Denny (Communications Lead).
+
+The content should be in a markdown file in the
+[src/content/posts](https://github.com/pyladies/global-conference/tree/main/src/content/posts)
+directory.
+
+You can see other as an example,
+but the most important thing is to include the following information
+at the beginning of the file:
+
+```
+---
+title: "Your title"
+meta_title: "Your meta title"
+description: "Short description of the post"
+date: 2025-06-02T05:00:00Z
+categories: ["Blog Post",]
+image: "/images/posts/default2025.png"
+authors: ["YourName"]
```

Comment:
Can you add clarification on how to add their blog author page? Eg having to create their own .md file under the /authors/ directory.